src/components/imageRe.re

```diff
@@ -0,0 +1,134 @@
+open Utils;
+
+external image : ReactRe.reactClass = "Image" [@@bs.module "react-native"];
+
+type imageURISource;
+
+external imageURISource :
+  uri::string =>
+  bundle::string? =>
+  method::string? =>
+  headers::list (string, string)? =>
+  body::string? =>
+  cache::string? =>
+  scale::float? =>
+  width::float =>
+  height::float =>
+  unit =>
+  imageURISource =
+  "" [@@bs.obj];
+
+type imageSource 'a =
+  | URI imageURISource
+  | Require 'a
+  | Multiple (list imageURISource);
+
+type defaultURISource;
+
+external defaultURISource :
+  uri::string => scale::float? => width::float => height::float => defaultURISource =
+  "" [@@bs.obj];
+
+type defaultSource 'a =
+  | URI defaultURISource
+  | Require 'a;
+
+type rawImageSourceJS;
+
+external rawImageSourceJS : 'a => rawImageSourceJS = "";
+
+module Event = {
+  type t;
+  type error;
+  type progress = {loaded: float, total: float};
+  external progress : t => progress = "nativeEvent" [@@bs.get];
```

Comment:
I think this doesn't work, `progress` would have to be `Js.t {. loaded: float, total: float }`